bin/tictactoe

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+
+puts "Welcome to Tic Tac Toe!"
+
+# Sets up game using lib/new_game.rb
+new_game
+
+puts "Do you want to play again? Y/N "
+answer = gets.strip.upcase
+
+until answer == "NO" || answer == "N"
+	new_game
+	puts "Do you want to play again? Y/N "
+	answer = gets.strip.upcase
+end
+
+puts "I hope you enjoyed playing Tic Tac Toe."
```

Comment:
Prefer to encapsulate CLI logic into a class. For example the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in call.